spec/system/dashboards/user_visits_dashboard_spec.rb

```diff
@@ -1,57 +1,137 @@
 require "rails_helper"
 
 RSpec.describe "Dashboard", type: :system, js: true do
-  let(:user) { create(:user) }
-  let(:listing) { create(:listing) }
+  let(:tag) { create(:tag) }
+  let(:organization) { create(:organization) }
+  let(:podcast) { create(:podcast) }
+  let(:listing) { create(:listing, user: collection.user) }
   let(:collection) { create(:collection, :with_articles) }
-  let(:collection_user) { collection.user }
+  let(:user1) { collection.user }
+  let(:user2) { create(:user) }
 
   context "when looking at analytics counters" do
     before do
-      sign_in user
+      sign_in user1
     end
 
-    it "shows the count of unspent credits" do
-      Credit.add_to(user, 2)
+    it "shows the count of unspent credits and listings created" do
+      Credit.add_to(user1, 2)
 
       Credits::Buy.call(
-        purchaser: user,
+        purchaser: user1,
         purchase: listing,
         cost: 1,
       )
       Credit.counter_culture_fix_counts
-      user.reload
+      user1.reload
 
       visit dashboard_path
 
       within "main#main-content > header" do
         expect(page).to have_text(/1\nCredits available/)
+        expect(page).to have_text(/1\nListings created/)
       end
     end
   end
 
   context "when looking at actions panel" do
     before do
-      sign_in collection_user
+      stub_const(
+        "DASHBOARD_PATHS",
```

Comment:
Why is this stubbed? It looks like it could just be a normal constant in this test class. Unless I missed something - you're not stubbing a constant in the subject under test, you're declaring one here for the other examples to re-use?

Reply:
When I tried that, it screamed at me “stub this constant!!!” 😆